### ISSUE TITLE
Allow to turn off row total in pivot table

### ISSUE THREAD

**Issue by liangsieng** [metabase/metabase#14979]
**Is your feature request related to a problem? Please describe.**
When pivot with more than 1 metric it will automatic have "row total" at the right end and sum up the columns value. Some of the cases do not required to sum up

![image](https://user-images.githubusercontent.com/38716433/109241620-99cd9c00-7814-11eb-8246-61500f15a8d9.png)

**Describe the solution you'd like**
To have option to turn off the "row total" 

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by disse70** [comment]
It will be very appreciate to have the option to exlude : 
   
   row totals 
   coloumn total

in a pivot table

**Comment by MartinCura** [comment]
Even better if we could format each column independently. For example, we have some columns with currencies which we want to add up and others with ratios where only averaging them makes sense.

**Comment by ginchauspe** [comment]
There is a workaround for this problem:
1) In the editor add a custom column right before the "Summarize" step with field formula concat(" ", " ") and any name you want.
2) Add this fake column in the right side of the "Summarize" step.
3) Go to Visualize -> Settings and add this column as the first one in the "columns" section.
Done! you will notice a new empty row on top of your columns section and the extra "Row Totals" will be gone. Hope it helps!

**Comment by Audrey-kemi** [comment]
> 

I tried this and it didn't work as expected.
I did notice a new empty row on top of my columns section but the extra "Row Totals" remained at the bottom.

**Comment by a-meledin** [comment]
> There is a workaround for this problem:
> 
> 1. In the editor add a custom column right before the "Summarize" step with field formula concat(" ", " ") and any name you want.
> 2. Add this fake column in the right side of the "Summarize" step.
> 3. Go to Visualize -> Settings and add this column as the first one in the "columns" section.
>    Done! you will notice a new empty row on top of your columns section and the extra "Row Totals" will be gone. Hope it helps!

Works on 44.1 if follow steps precisely.

**Comment by LukaNeg** [comment]
> I tried this and it didn't work as expected.
I did notice a new empty row on top of my columns section but the extra "Row Totals" remained at the bottom.

Same problem. Upgraded to the latest version. I followed all the steps and the column and row totals disappear at first. But then I save the question and the column totals appear back. Row totals are still removed, but it's the column totals (the row on the bottom) that I really want to get rid of. 

Anyone find any other work-arounds?
